### ISSUE TITLE
Phased hand off of PDF design to engineering

### ISSUE THREAD

**Issue by carodew** [department-of-veterans-affairs/abd-vro#608]
### 

Coordinate through @amylai-nava on hand off. 

### Design changes
- [x] visual design
- [x] BP table (minus deduplication)
- [x] Swapping the order of columns to separate the dates
- [x] Design empty state for (dig up previous designs in Google Drive)
    - [x] no service keywords found
    - [x] no diagnoses found
    - [x] no medications found
    - [x] no other medical evidence found
- [x] not all relevant documents were able to be scanned 
- [x] rules for tables breaking across pages

### Activities

- [x] Discuss with PEZ about timeline for approvals. 
- [x] Dig up [print quality VA logo](https://vaww.va.gov/6102/graphics_resource_center.asp) (Need to be on VA network to access)

### Open questions

- [x] What parts of the PDF design are we approved to do? (p clear that whatever MAS agreed to we can do, everything else is unclear) 
    - waiting on some decisions of the higher level approach questions from leadership, but anything MAS agreed to we feel we can go ahead and do
- [x] How much documentation do the engineers want? 
- [x] Presumptive service connection? 
- [x] Copy changes?  captured in #607 

### Related work
- #448 
- #632 Presumptive section variations 
   - (if we don't find a flash, what if we find two flashes that are relevant, if a condition can be covered by two different cohorts, what if there's more than 1 DC code for the condition). Do/don't have service. (note we're only creating this section if we find the AO flash). This is only if we needed to prove it out.

**Comment by carodew** [comment]
## MAS comments summary 

### Presumptive service connection
- does not account for all types of service connection (wording may be too leading towards presumptive SC than other types)
- Veteran can FOIA eFolder and anything that says presumptive can be legally binding
- DX could be legally binding, too close to making a medical determination
- not all vets have VAMC data
- wording of OCRed source docs should match VBMS (not sure why?)
- Do not show location data for Lighthouse data bc it would be too difficult to do the same accurately with OCR data
- deduping is risky
- duplication in about this document language. 
- should not suggest what adjudicator does with the document / evidence (second sentence) 
- should start with "this document summarizes" 
- Reword: OCR does not look at all documents submitted with the claim or in eFolder, only docs that are likely to contain medical or service information 
- add language that says records receive the same day as the claim may not have been OCR'd. 

### Agreed 
- formatting changes for BP readings, date separation, and removing BP prefix
- visual design changes

**Comment by carodew** [comment]
[Design doc](https://docs.google.com/document/d/1RAu4IW8N9OtBVsLh9InOu3pSsUnqa-sgHV99C9CGDu0/edit?usp=sharing)

[Presumptive service variations ](https://docs.google.com/document/d/1fiXTfOqSno_dZstlt9M8o5X-zdoq3SOXpL43vVmJBQM/edit#)

**Comment by carodew** [comment]
11/17: Provided interim type styles here: https://dsva.slack.com/archives/C01Q7979Z7D/p1668715983957409?thread_ts=1668455287.286959&cid=C01Q7979Z7D 

11/17:  Also provided visual design feedback here: https://dsva.slack.com/archives/C01Q7979Z7D/p1668718311603469?thread_ts=1668455287.286959&cid=C01Q7979Z7D

**Comment by carodew** [comment]
@amylai-nava I'm thinking we can call this one 'done' for now, knowing that we'll pick this back up again (under a new ticket) in January when we're testing with actual data. Thoughts?

**Comment by amylai-nava** [comment]
@carodew I agree!
